tests/pyfunc/test_chat_model.py

```diff
@@ -380,3 +380,23 @@ def test_chat_model_predict_stream(tmp_path):
 
     response = next(loaded_model.predict_stream({"messages": messages}))
     assert response["choices"][0]["message"]["content"] == json.dumps(messages)
+
+
+# test that users cannot overwrite the 'object' field in ChatResponse
+def test_chat_model_response_cannot_overwrite_object():
+    message = ChatMessage(role="user", content="Hello!")
+    params = ChatParams(**DEFAULT_PARAMS)
+    mock_response = get_mock_response([message], params)
+
```

Comment:
Can we add test case for setting invalid value in constructor?
```
ChatResponse(..., object="invalid")
```
This should raise.

Reply:
good suggestion. feedback incorporated